apps/labrinth/src/routes/internal/billing.rs

```diff
@@ -377,6 +380,9 @@ pub async fn edit_subscription(
 
             if *cancelled {
                 open_charge.status = ChargeStatus::Cancelled;
+            } else if open_charge.status == ChargeStatus::Failed {
+                // Force another resubscription attempt
+                open_charge.last_attempt = Some(Utc::now() - Duration::days(2));
```

Comment:
It's a bit hacky to mangle the last charge attempt to get the side effect of it being retried, but there isn't a better alternative, so let's do this.

Reply:
I was thinking about it more - it could be possible to add a new column with